src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,117 @@
+package Duke;
+
+import Duke.Exceptions.EmptyMessageException;
+import Duke.Exceptions.WrongCommandException;
+import Duke.Exceptions.WrongDateFormatException;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Driver class of Duke
+ */
+public class Duke {
+
+    static ListStorage myListStorage = new ListStorage();
+    static Printer myPrinter = new Printer();
+    static Disk myDisk;
+
+    /*
+     * Empty constructor for Duke
+     */
+    public Duke() {
+        myDisk = new Disk("src/main/java/data/savedTasks.txt", myListStorage);
+        try {
+            myDisk.loadFromDisk();
+        } catch (IOException e) {
+            myPrinter.printExceptions(e);
+        }
+    }
+
+    /**
+     * Constructor for Duke
+     * Loads existing ListStorage in Disk
+     * @param filePath filePath
+     */
+    public Duke(String filePath) {
+        myDisk = new Disk(filePath, myListStorage);
+        try {
+            myDisk.loadFromDisk();
+        } catch (IOException e) {
+            myPrinter.printExceptions(e);
+        }
+    }
+
+    public static void main(String[] args) {
+        new Duke("src/main/java/data/savedTasks.txt").dukeRunner();
+    }
+
+    /*
+     * Runner method for Duke
+     */
+    public void dukeRunner() {
+        myPrinter.printGreeting();
+        while(true){
+            try{
+                parseCommand();
+                break;
+            } catch (WrongCommandException e) {
+                myPrinter.printExceptions(e);
+            }
```

Comment:
If I'm not wrong, this is to ensure that Duke continues to run even after accepting a wrong command? I like the logic of this portion.